### PR TITLE
[ETS-521] Cloudwatch metrics and alarms to slack via SNS

### PR DESCRIPTION
**Pull request by vyasworks**

### CODE REVIEW

**Review by hitankar-freshworks — COMMENTED** [review]
Incredible work. However, I have some suggestions to make this really usable and make the codebase more manageable. 
- I suggest we move the code for handling SNS event from terraform folder to server folder. I think there is great potential for our main code to use this as a service to send messages to `slack`. 
- Instead of our application waiting on a promise from slack to send a message, it can push to SNS and forget it.

File: terraform/service_acc.tf [hunk 1]
```diff
@@ -39,6 +39,6 @@ data "aws_iam_policy_document" "service_account" {
 }
 
 
-output "service_account_iam" {
-  value = data.aws_iam_policy_document.service_account.json
-}
+# output "service_account_iam" {
```
Comment (hitankar-freshworks) [inline terraform/service_acc.tf:42]: lets not bloat our codebase with commented code

**Review by sunghwan-park-fw — DISMISSED** [review]

File: terraform/service_acc.tf (hunk 1 above)
Comment (vyasworks) [reply terraform/service_acc.tf:43]: Added comment to reason why

**Comment by vyasworks** [comment]
> Incredible work. However, I have some suggestions to make this really usable and make the codebase more manageable.
> 
> * I suggest we move the code for handling SNS event from terraform folder to server folder. I think there is great potential for our main code to use this as a service to send messages to `slack`.
> * Instead of our application waiting on a promise from slack to send a message, it can push to SNS and forget it.

moved to root with its own folder

**Review by sunghwan-park-fw — APPROVED** [review]

**Pull request merged**